### ISSUE TITLE
 field gitlab not found in type config.Release

### ISSUE THREAD

**Issue by fitzix** [goreleaser/goreleaser#1067]
**Describe the bug**

According to the documentation https://goreleaser.com/customization/#Release , gitlab in yaml will unmarshal error

**To Reproduce**

```yaml
release:
  # Same as for github
  # Note: it can only be one: either github or gitlab
  gitlab:
    owner: user
    name: repo

  # You can change the name of the GitLab release.
  # Default is `{{.Tag}}`
  name_template: "{{.ProjectName}}-v{{.Version}} {{.Env.USER}}"

  # You can disable this pipe in order to not upload any artifacts to
  # GitLab.
  # Defaults to false.
  disable: true
```

**Expected behavior**

**Environment (please complete the following information):**

- OS: mac
- GoReleaser Version
version: 0.111.0
commit: 1cf34166b6128c5348ff75e6923d715251ab8c85
built at: 2019-06-28T12:21:37Z
built by: goreleaser

**Additional context**

```

   • releasing using goreleaser 0.111.0...
   • loading config file       file=.goreleaser.yml
   ⨯ release failed after 0.00s error=yaml: unmarshal errors:
  line 19: field gitlab not found in type config.Release

```

**Comment by caarlos0** [comment]
sorry, this is on master only... 

need to improve those docs at some point

**Comment by caarlos0** [comment]
This should be fixed now. The docs thing will be another (big) thing.

Gitlab support was released into v0.112.0 yesterday :)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by caarlos0** [comment]
Derek close

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
